db/schema.rb

```diff
@@ -18,6 +18,11 @@
   enable_extension "pgcrypto"
   enable_extension "plpgsql"
 
+  # Custom types defined in this database.
+  # Note that some types may not work with other database engines. Be careful if changing database.
+  create_enum "future_interest_type", ["yes", "no", "unknown"]
+  create_enum "trigger_type", ["provider", "trainee"]
```

Comment:
Should this be here?

Reply:
I think the comments are automatically generated